docs/data-ingestion/Prometheus.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Prometheus
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "prometheus"
+    parent: "data-ingestion"
+    weight: 1
+aliases:
+  - /data-ingestion/prometheus.html
+  - /data-ingestion/Prometheus.html
+---
+
+VictoriaMetrics Supports ingesting data from Prometheus via the Prometheus remote write protocol.
+The documentation for pushing data from Prometheus to VictoriaMetrics is located here](https://docs.victoriametrics.com/#prometheus-setup) in order to maintain backward compatibility existing documentation.
```

Comment:
I think it is better to do it vice-versa - link from README.MD to these docs.
But I think this could require better content organization.

@func25 can I ask you for help here? wdyt about creating a section `Integrations` or something like this, where we'll list docs related to Prometheus, Grafana, various agents and monitoring solutions. If organized well, it could help to keep all related connectivity info in one place and offload it from https://docs.victoriametrics.com/single-server-victoriametrics/ page. 

This might be out of scope for this PR, but it is better to do it earlier than later